common/changes/@itwin/presentation-backend/presentation-OpenTelemetry_2022-08-16-09-01.json

```diff
@@ -2,7 +2,7 @@
   "changes": [
     {
       "packageName": "@itwin/presentation-backend",
-      "comment": "",
+      "comment": "Added `PresentationManagerProps.diagnosticsCallback` property for setting up OpenTelemetry reporting.",
```

Comment:
The changes in this package aren't tied to OpenTelemetry. I'd replace that term with "diagnostics".